test_barcode_docker_run.sh

```diff
@@ -7,11 +7,11 @@
 
 # test case 1: barcode missing parameter
 echo "test case 1: barcode missing parameter"
-./docker-run.sh --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0 --ocr 5 CPU --barcode
+./run.sh --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0 --ocr 5 CPU --barcode
```

Comment:
barcode requires an argument, maybe you can add 5 at the end

Reply:
i am not changing the use case for this old code, you can open another issue for this if you want